main/test/src/util/TestEvaluator.scala

```diff
@@ -12,12 +12,12 @@ object TestEvaluator{
   val externalOutPath = os.pwd / 'target / 'external
 
 
-  def static(module: TestUtil.BaseModule)(implicit fullName: sourcecode.FullName) = {
+  def static(module: => TestUtil.BaseModule)(implicit fullName: sourcecode.FullName) = {
```

Comment:
This is needed to accommodate a cyclic reference  (`BloopImpl` requires an evaluator contains a path-dependant trait )